pom.xml

```diff
@@ -344,6 +345,45 @@ limitations under the License.
                 </executions>
             </plugin>
 
+            <!-- Check for banned dependencies -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
+                <executions>
+                    <execution>
+                        <id>enforce-banned-dependencies</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <bannedDependencies>
+                                    <excludes>
+                                        <exclude>commons-logging:commons-logging</exclude>
+                                        <exclude>log4j:log4j</exclude>
+                                        <exclude>org.apache.logging.log4j:log4j-core</exclude>
+                                        <exclude>org.apache.logging.log4j:log4j-slf4j-impl</exclude>
+                                        <exclude>org.jboss.logmanager:jboss-logmanager</exclude>
+                                        <exclude>org.jboss.logmanager:log4j-jboss-logmanager</exclude>
+                                        <exclude>org.jboss.slf4j:slf4j-jboss-logmanager</exclude>
+                                        <exclude>org.slf4j:slf4j-jcl</exclude>
+                                        <exclude>org.slf4j:slf4j-jdk14</exclude>
+                                        <exclude>org.slf4j:slf4j-log4j12</exclude>
+                                        <exclude>org.slf4j:slf4j-log4j13</exclude>
+                                        <exclude>org.slf4j:slf4j-nop</exclude>
+                                        <exclude>org.slf4j:slf4j-simple</exclude>
```

Comment:
I think this is what Dermot is looking for, add this to the includes section and it will override excludes.
org.slf4j:slf4j-simple:*:*:test

Reply:
@andyreidz Had a go testing it there, this appears to be the syntax for it org.slf4j:slf4j-simple:*.*:jar:test. Looks like you need to wildcard empty fields. I had a test of it there and this seems to give the desired result. If the dependency is added with compile or runtime scope the build fails. If its test scope the build continues.